logcheck/testdata/src/onlyAllowContextual/onlyAllowContextual.go

```diff
@@ -26,7 +26,9 @@ import (
 )
 
 func doNotAlllowKlog() {
-	klog.InfoS("test log")       // want `function "InfoS" should not be used, convert to contextual logging`
-	klog.ErrorS(nil, "test log") // want `function "ErrorS" should not be used, convert to contextual logging`
-	klog.V(1).Infof("test log")  // want `function "V" should not be used, convert to contextual logging` `function "Infof" should not be used, convert to contextual logging`
+	klog.InfoS("test log")                         // want `function "InfoS" should not be used, convert to contextual logging`
+	klog.ErrorS(nil, "test log")                   // want `function "ErrorS" should not be used, convert to contextual logging`
+	klog.V(1).Infof("test log")                    // want `function "V" should not be used, convert to contextual logging` `function "Infof" should not be used, convert to contextual logging`
+	klog.KObjs(nil)                                // want `Detected usage of deprecated helper "KObjs". Please switch to "KObjSlice" instead.`
+	klog.InfoS("test log", "key", klog.KObjs(nil)) // want `function "InfoS" should not be used, convert to contextual logging` `Detected usage of deprecated helper "KObjs". Please switch to "KObjSlice" instead.`
```

Comment:
Can you add `klog.KObjs` at the bottom, with one blank line as separator? Then we don't need to reformat all of the other lines and the patch becomes smaller.

Reply:
/Ack.